### ISSUE TITLE
Peril specific reinsurance terms

### ISSUE THREAD

**Issue by johcarter** [OasisLMF/ReinsuranceTestTool#58]
Need to test with a real model

**Comment by mpinkerton-oasis** [comment]
This could also be added to test tool in the first instance, with an extra command flag to specify which perils/sub-perils should generate loss. Essentially we just need to link to the correct items, right. (I think any new functionality should be added to the tool and tested, and then integrated into the platform)

**Comment by johcarter** [comment]
yes that would work. would we need a new item file, with losses for 2 perils and the same 4 locations?

**Comment by mpinkerton-oasis** [comment]
I think the code should generate the items file, based on the locations. We could have a widget where you choose the perils and damage factors which would create appropriate items.
